app/templates/eventView.html

```diff
@@ -37,6 +37,9 @@ <h1 id="title">{{page_title}}</h1>
             <div class="row">
                 <div class="col-md-6">
                     <h3><br>Description</h3><p>{{eventData.description|urlize}}</p><br>
+                    {% if eventData.foodProvided %}
+                        <p>This event provides food.</p>
```

Comment:
Opposite of my comment above, I think it might look better if we change the wording here so it is not much like the other statements. Maybe something like "Food will be provided at this event." or "There will be food provided at this event."